src/app/pages/HomePage/GraphTooltip/GraphTooltipMobile.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC, PropsWithChildren } from 'react'
+import Fade from '@mui/material/Fade'
+import CloseIcon from '@mui/icons-material/Close'
+import { COLORS } from '../../../../styles/theme/colors'
+import { useTranslation } from 'react-i18next'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+
+export const MobileBackdrop = styled(Box)(() => ({
+  position: 'fixed',
+  inset: 0,
+  backgroundColor: COLORS.black,
+  opacity: 0.3,
+  zIndex: 4,
```

Comment:
This value relies on zIndex that is set in another place. It's probably time to create zIndexWhatever somewhere in home page and export it.